src/lib/ngsi/EntityIdVector.cpp

```diff
@@ -193,13 +193,18 @@ EntityId* EntityIdVector::get(int ix)
   return vec[ix];
 }
 
+const EntityId* EntityIdVector::get(int ix) const
+{
```

Comment:
I think there is an issue about implement [] in this kind of classes, but not fully implement, so probably we are in a situation in which some classes use it, but no others. Thus, I think this PR (or others in this "saga") shouldn't go into that.